sdk/CMakeLists.txt

```diff
@@ -25,9 +25,7 @@ if(NOT WITH_LOGS_PREVIEW)
 endif()
 
 set(METRICS_EXCLUDE_PATTERN "")
-if(NOT WITH_METRICS_PREVIEW)
-  set(METRICS_EXCLUDE_PATTERN "_metrics")
-endif()
+set(METRICS_EXCLUDE_PATTERN "_metrics")
```

Comment:
Once `_metrics` are removed (PR #1734), `METRICS_EXCLUDE_PATTERN` can probably be removed as well.

Reply:
thanks, removed